Readability.js

```diff
@@ -524,7 +529,8 @@ Readability.prototype = {
         if (stripUnlikelyCandidates) {
           if (this.REGEXPS.unlikelyCandidates.test(matchString) &&
               !this.REGEXPS.okMaybeItsACandidate.test(matchString) &&
-              node.tagName !== "BODY") {
+              node.tagName !== "BODY" &&
+              node.tagName !== "A") {
```

Comment:
Is this addition really necessary? It seems unlikely to really impact candidate selection (which is mostly paragraph-related anyway, I think?).